docs/src/troubleshooting.md

```diff
@@ -139,3 +139,8 @@ begin
 end
 ```
 
+
+## Distribution of cells on multiple MPI ranks in [`P4estMesh`](@ref)
+
+The [`P4estMesh`](@ref) allows coarsening the mesh by default. When Trixi is parallelized with multiple MPI ranks this has the consequence that families of cells are kept on the same MPI rank to be able to coarsen them easily. This might cause an unbalanced distribution of cells on different ranks. For 2D meshes this also means that each rank will at least own 4 cells and for 3D meshes each rank will at least own 8 cells.
```

Comment:
```suggestion
The [`P4estMesh`](@ref) allows coarsening the mesh by default. When Trixi is parallelized with multiple MPI,
ranks this has the consequence that sibling cells (i.e., child cells with the same parent cell)
are kept on the same MPI rank to be able to coarsen them easily. This might cause an unbalanced
distribution of cells on different ranks.
```
I rephrased your statement a little bit to clarify what a "family" means. I also removed the last sentence since I believe it is incorrect: When there are 4 cells left on a rank, you can still coarsen it to one cell, can't you?

Reply:
Thanks, @sloede. Yes, of course you are right. I wanted to say that **initially** each rank will get at least 4 (8) cells to be able to coarsen them. I think this could be helpful to know because you might wonder (e.g., by looking at the output of the summary callback at the beginning) why the root rank has 4 cells when you expect it to have less because you know that the mesh is small and you use many ranks. You might recognize this even without recognizing that other ranks got 0 cells.

Reply:
Thanks for the clarification. I modified the suggestion above as per your comment.

Reply:
Thanks!